[main] `Ubuntu2404Arm64_Offline` and `Alpine320_Online` are building failed with prebuilt error

[Failing build](https://dev.azure.com/dnceng/internal/_build/results?buildId=2577788&view=logs&j=b1caa8c7-837c-5a0e-fa2b-360b4dfa6cb4&t=5a647af6-2fef-58c2-4caf-850dc1bc163c) (internal Microsoft link)

`Alpine320_Online_MsftSdk_x64` building failed with `error: 4 Prebuilts Exist`

      Packaged sdk symbols in '/vmr/artifacts/assets/Release/dotnet-symbols-sdk-10.0.100-alpha.1.24558.1-alpine.3.20-x64.tar.gz'
      Found 4 files in prebuilt packages dir.
      Tarball '/vmr/artifacts/assets/Release/Private.SourceBuilt.Prebuilts.10.0.100-alpha.1.24558.1.alpine.3.20-x64.tar.gz' was successfully created from '/vmr/artifacts/prebuilt-report/prebuilt-packages/'
    /vmr/eng/finish-source-only.proj(180,5): error : 4 Prebuilts Exist

[generated-new-baseline.xml](https://artprodcus3.artifacts.visualstudio.com/Ab55de4ed-4b5a-4215-a8e4-0a0a5f71e7d8/7ea9116e-9fac-403d-b258-b31fcf1bb293/_apis/artifact/cGlwZWxpbmVhcnRpZmFjdDovL2RuY2VuZy9wcm9qZWN0SWQvN2VhOTExNmUtOWZhYy00MDNkLWIyNTgtYjMxZmNmMWJiMjkzL2J1aWxkSWQvMjU3Nzc4OC9hcnRpZmFjdE5hbWUvQWxwaW5lMzIwX09ubGluZV9Nc2Z0U2RrX3g2NF9CdWlsZExvZ3NfQXR0ZW1wdDE1/content?format=file&subPath=%2Fartifacts%2Fprebuilt-report%2Fgenerated-new-baseline.xml)

    <NeverRestoredTarballPrebuilts>
      <PackageIdentity Id="Microsoft.AspNetCore.App.Runtime.linux-x64" Version="9.0.0-rc.2.24474.3"/>
      <PackageIdentity Id="Microsoft.NETCore.App.Crossgen2.linux-x64" Version="9.0.0-rc.2.24473.5"/>
      <PackageIdentity Id="Microsoft.NETCore.App.Host.linux-x64" Version="9.0.0-rc.2.24473.5"/>
      <PackageIdentity Id="Microsoft.NETCore.App.Runtime.linux-x64" Version="9.0.0-rc.2.24473.5"/>
    </NeverRestoredTarballPrebuilts>
    

`Ubuntu2404Arm64_Offline_MsftSdk_arm64` building failed with `error NU1102` and there are 4 repos failed: `fsharp`, `sourcelink`, `scenario-tests` and `vstest`

      error NU1102: Unable to find package Microsoft.NETCore.App.Runtime.linux-arm64 with version (= 9.0.0-rc.2.24473.5) 
      error NU1102: - Found 1 version(s) in previously-source-built [ Nearest version: 10.0.0-alpha.1.24554.9 ] 
      error NU1102: - Versions from /vmr/.dotnet/sdk/10.0.100-alpha.1.24555.54/FSharp/library-packs were not considered 
      error NU1102: - Versions from prebuilt were not considered 
      error NU1102: - Versions from reference-packages were not considered 
      error NU1102: - Versions from source-built-transport-arcade were not considered 
      error NU1102: Unable to find package Microsoft.NETCore.App.Crossgen2.linux-arm64 with version (= 9.0.0-rc.2.24473.5) 
      error NU1102: - Found 1 version(s) in previously-source-built [ Nearest version: 10.0.0-alpha.1.24554.9 ] 
      error NU1102: - Versions from /vmr/.dotnet/sdk/10.0.100-alpha.1.24555.54/FSharp/library-packs were not considered 
      error NU1102: - Versions from prebuilt were not considered 
      error NU1102: - Versions from reference-packages were not considered 
      error NU1102: - Versions from source-built-transport-arcade were not considered 
      error NU1102: Unable to find package Microsoft.NETCore.App.Host.linux-arm64 with version (= 9.0.0-rc.2.24473.5) 
      error NU1102: - Found 1 version(s) in previously-source-built [ Nearest version: 10.0.0-alpha.1.24554.9 ] 
      error NU1102: - Versions from /vmr/.dotnet/sdk/10.0.100-alpha.1.24555.54/FSharp/library-packs were not considered 
      error NU1102: - Versions from prebuilt were not considered 
      error NU1102: - Versions from reference-packages were not considered 
      error NU1102: - Versions from source-built-transport-arcade were not considered 
      error NU1102: Unable to find package Microsoft.AspNetCore.App.Runtime.linux-arm64 with version (= 9.0.0-rc.2.24474.3) 
      error NU1102: - Found 1 version(s) in previously-source-built [ Nearest version: 10.0.0-alpha.2.24555.17 ] 
      error NU1102: - Versions from /vmr/.dotnet/sdk/10.0.100-alpha.1.24555.54/FSharp/library-packs were not considered 
      error NU1102: - Versions from prebuilt were not considered 
      error NU1102: - Versions from reference-packages were not considered 
      error NU1102: - Versions from source-built-transport-arcade were not considered

--- a/src/SourceBuild/content/eng/Versions.props
+++ b/src/SourceBuild/content/eng/Versions.props
@@ -25,7 +25,7 @@
     -->
     <PrivateSourceBuiltSdkVersion>10.0.100-alpha.1.24556.1</PrivateSourceBuiltSdkVersion>
     <PrivateSourceBuiltArtifactsVersion>10.0.100-alpha.1.24556.1</PrivateSourceBuiltArtifactsVersion>
-    <PrivateSourceBuiltPrebuiltsVersion>0.1.0-10.0.100-5</PrivateSourceBuiltPrebuiltsVersion>
+    <PrivateSourceBuiltPrebuiltsVersion>0.1.0-10.0.100-6</PrivateSourceBuiltPrebuiltsVersion>
     <!-- command-line-api dependencies -->
     <SystemCommandLineVersion>2.0.0-beta4.24126.1</SystemCommandLineVersion>
     <!-- msbuild dependencies -->

